torch_brain/utils/readout.py

```diff
@@ -77,7 +77,7 @@ def prepare_for_readout(
 
     # resolve eval mask
     eval_mask = None
-    eval_interval_key = data.config.get("eval_interval", None)
```

Comment:
This is off-topic from the PR description. But seems like a bug to me, so I fixed it here.